ios/browser/ui/webui/ai_chat/ai_chat_untrusted_conversation_ui.mm

```diff
@@ -85,6 +85,10 @@ void OpenURLFromResponse(const GURL& url) override {
     OpenURL(url);
   }
 
+  void AddTabToThumbnailTracker(int32_t tab_id) override {}
+
+  void RemoveTabFromThumbnailTracker(int32_t tab_id) override {}
+
```

Comment:
Can you add a comment that just says these aren't supported on mobile or something

Reply:
done
```cpp
  // No current thumbnail tracker need or support on iOS
  void AddTabToThumbnailTracker(int32_t tab_id) override {}
  void RemoveTabFromThumbnailTracker(int32_t tab_id) override {}
```